app_dart/lib/src/request_handling/subscription_handler.dart

```diff
@@ -138,8 +138,8 @@ abstract class SubscriptionHandler extends RequestHandler<Body> {
     await runZoned<Future<void>>(
       () async => await super.service(
         request,
-        onError: (_) async {
-          log.warning('Failed to process $message');
+        onError: (exception) async {
```

Comment:
Specify the type

https://dart-lang.github.io/linter/lints/always_specify_types.html

Reply:
Added a type - I also added it to `analysis_options.yaml` if that's okay

Reply:
Nevermind - It looks like there are quite a few unspecified types, I don't want to clutter this PR too much, so I can add it in an additional PR